### PR TITLE
Pass description in sections

### PR DESCRIPTION
**Pull request by SaraVieira**
In the project I am currently in we want to show section descriptions in the header of the section.

Also changed one of the styleguide configs to show this

Is this okay ?

### CODE REVIEW

**Comment by SaraVieira** [comment]
docs changed

File: src/rsg-components/Section/SectionRenderer.js [hunk 1]
```diff
@@ -10,14 +11,16 @@ const styles = ({ space }) => ({
...
+	const { classes, name, slug, content, components, sections, depth, description } = allProps;
+
 	return (
 		<section className={classes.root}>
 			{name && (
 				<SectionHeading depth={depth} id={slug} slotName="sectionToolbar" slotProps={allProps}>
 					{name}
 				</SectionHeading>
 			)}
+			{description && <Markdown text={`${description}`} inline />}
```
Comment (sapegin) [inline src/rsg-components/Section/SectionRenderer.js]: 1. Why is it inline? Shouldn’t it be rendered on its own line with a proper whitespace below?
2. I think just `text={description}` will be enough.

File: src/rsg-components/Section/Section.js [hunk 2]
```diff
@@ -7,14 +7,15 @@ import SectionRenderer from 'rsg-components/Section/SectionRenderer';
...
-	const { name, slug, content, components, sections } = section;
+	const { name, slug, content, components, sections, description } = section;
 
 	const contentJsx = content && <Examples examples={content} name={name} />;
 	const componentsJsx = components && <Components components={components} depth={depth + 1} />;
 	const sectionsJsx = sections && <Sections sections={sections} depth={depth + 1} />;
 
 	return (
 		<SectionRenderer
+			description={description}
```
Comment (sapegin) [inline src/rsg-components/Section/Section.js]: I think you need to update PropTypes of SectionRenderer.

Comment (SaraVieira) [reply src/rsg-components/Section/Section.js]: Done , sorry :)

File: src/rsg-components/Section/SectionRenderer.js (hunk 1 above)
Comment (SaraVieira) [reply src/rsg-components/Section/SectionRenderer.js]: Yor are right about both and changed :)

**Comment by SaraVieira** [comment]
So we got them Node 4 failing again :(

**Comment by sapegin** [comment]
Try to merge master, I’ve made it optional.

**Comment by SaraVieira** [comment]
Passed 😍

**Comment by sapegin** [comment]
:shipit:

**Pull request merged**